scripts/microbe_stage/patch_map_generator.as

```diff
@@ -11,15 +11,26 @@ PatchMap@ runGeneration()
 {
     PatchMap@ map = PatchMap();
 
-    Patch@ patch1 = Patch("Pangonian vents", 0, getBiomeTemplate("aavolcanic_vent"));
-
+    Patch@ patch0 = Patch("Pangonian vents", 0, getBiomeTemplate("aavolcanic_vent"));
     auto defaultSpecies = Species::createDefaultSpecies();
-
     for(uint i = 0; i < defaultSpecies.length(); ++i){
-        patch1.addSpecies(defaultSpecies[i]);
+        patch0.addSpecies(defaultSpecies[i]);
     }
+    patch0.setScreenCoordinates(Float2(100,100));
+
+    Patch@ patch1 = Patch("Pangonaian Mesopelagic", 1, getBiomeTemplate("mesopelagic"));
+    patch1.setScreenCoordinates(Float2(200,100));
+    patch1.addNeighbour(0);
+    patch0.addNeighbour(1);
+
+    Patch@ patch2 = Patch("Pangonian Epipelagic", 2, getBiomeTemplate("default"));
+    patch2.setScreenCoordinates(Float2(200,200));
```

Comment:
I think the coordinates should be different for the different patches.

Reply:
I thought I did put the patches in a triangle with different coordinates for each patch.

Reply:
You are correct. I misread one of the numbers.